Source/Android/app/src/main/res/layout-ldrtl/list_item_cheat.xml

```diff
@@ -1,37 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/root"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:focusable="true"
-    android:nextFocusLeft="@id/checkbox">
-
-    <TextView
-        android:id="@+id/text_name"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        style="@style/TextAppearance.MaterialComponents.Headline5"
-        android:textSize="16sp"
-        tools:text="Hyrule Field Speed Hack"
-        android:layout_margin="@dimen/spacing_large"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toStartOf="@id/checkbox"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent" />
-
-    <CheckBox
-        android:id="@+id/checkbox"
-        android:layout_width="48dp"
-        android:layout_height="64dp"
-        android:focusable="true"
-        android:gravity="center"
-        android:nextFocusRight="@id/root"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toEndOf="@id/text_name"
-        app:layout_constraintTop_toTopOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Why was this file deleted?

Note: This file swaps nextFocusLeft and nextFocusRight compared to the LTR file. If that's non-obvious, perhaps it would be nice to add a comment about it in the file?

Reply:
Ah I didn't notice it before. I added a comment.